Sources/AcknowList/AcknowListViewController.swift

```diff
@@ -441,25 +375,30 @@ open class AcknowListViewController: UITableViewController {
 
     /**
      Tells the delegate that the specified row is now selected.
-
-     - parameter tableView: A table-view object informing the delegate about the new row selection.
-     - parameter indexPath: An index path locating the new selected row in `tableView`.
+     - Parameters:
+        - tableView: A table-view object informing the delegate about the new row selection.
+        - indexPath: An index path locating the new selected row in `tableView`.
      */
     open override func tableView(_ tableView: UITableView, didSelectRowAt indexPath: IndexPath) {
         if let acknowledgement = acknowledgements[(indexPath as NSIndexPath).row] as Acknow?,
-           let navigationController = self.navigationController {
-            let viewController = AcknowViewController(acknowledgement: acknowledgement)
-            navigationController.pushViewController(viewController, animated: true)
+           let navigationController = navigationController {
+            if acknowledgement.text != nil {
+                let viewController = AcknowViewController(acknowledgement: acknowledgement)
+                navigationController.pushViewController(viewController, animated: true)
+            }
+            else if let repository = acknowledgement.repository {
+                let safariViewController = SFSafariViewController(url: repository)
```

Comment:
I am facing a crash on my private Swift packages, which are pulled in via _ssh_ rather than _https_. It causes the following error when initializing `SFSafariViewController` from `acknowledgement.repository` having a value like `"git@github.com:myuser/MyPackage.git"`. 

**Crash Log:**
```
*** Terminating app due to uncaught exception 'NSInvalidArgumentException', reason: 'The specified URL has an unsupported scheme. Only HTTP and HTTPS URLs are supported.'

*** First throw call stack:
(
	0   CoreFoundation                      0x000000010bcce604 __exceptionPreprocess + 242
	1   libobjc.A.dylib                     0x0000000109ceba45 objc_exception_throw + 48
	2   CoreFoundation                      0x000000010bcce4e2 -[NSException initWithCoder:] + 0
	3   SafariServices                      0x000000010b112cae -[SFSafariViewController initWithURL:configuration:] + 105
	4   SafariServices                      0x000000010b1132ae -[SFSafariViewController initWithURL:] + 66
	5   Countdown                           0x000000010907a431
```

Reply:
@martnst Thanks a lot for the feedback!

That’s a very good point. I’ve just updated the code to limit navigation to repositories with `http`/`https` URLs only.

Reply:
@vtourraine works fine for me now 😎